### PR TITLE
Wilke/quick start

### PR DESCRIPTION
**Pull request by wilke**
Initial set of documentation. A list of readme(s) for quick start, testing and examples.

### CODE REVIEW

**Comment by wilke** [comment]
closes #136

**Comment by wilke** [comment]
closes #167

**Comment by wilke** [comment]
closes #175

**Review by jameshcorbett — APPROVED** [review]
Looks good, a few minor comments below.

File: scripts/WORKFLOW-EXAMPLES.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
+# Job examples
```
Comment (andre-merzky) [inline scripts/WORKFLOW-EXAMPLES.md:1]: Is the `ALL_UPPERCASE.md` filename a convention for MD files somehow?  Don't care much either way but wondered about it...

**Review by andre-merzky — APPROVED** [review]
LGTM - left some minor comments.

File: scripts/pdf2wc4singularity.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+from psij import Export
+e = Export()
+
+jex = psij.JobExecutor.get_instance('local')
+
+N = 1  # number of jobs to run
+
+pdfFile = "example.pdf"
+pngFile = "worcloud.png"
+image = "docker://wilke/pdf2wordcloud:psij"
```
[the remark this answers is not in the document]
Comment (wilke) [reply scripts/pdf2wc4singularity.py:13]: @jameshcorbett Yes, it is public on docker hub. I prefer to move it somewhere project related. Any idea?

File: scripts/pdf2wc4singularity.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import os
+import psij
+# for debugging remove later
```
[the remark this answers is not in the document]
Comment (wilke) [reply scripts/pdf2wc4singularity.py]: Yes - removed now

File: QuickStart.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+3. follow the steps above
+
+
+
+
+
+## Hello world
+
+**Requirements**
+- python3.7
```
[the remark this answers is not in the document]
Comment (wilke) [reply QuickStart.md:33]: I only tested it with 3.7. Will check but shouldn't be a problem.

File: scripts/WORKFLOW-EXAMPLES.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y \
+  poppler-utils \
+  python3-pip
+RUN pip install wordcloud
+```
+
+## Executing a singularity conatiner
+
+We will run the PDF2Wordcloud example in a container. We will use the docker conatiner described above and execute it with singularity. In this case the spec defines the execution of a container with singularity. The actual command line tool is passed as an optional argument after all requiered singularity options.
```
[the remark this answers is not in the document]
Comment (wilke) [reply scripts/WORKFLOW-EXAMPLES.md:103]: Corrected typo, thanks.

File: scripts/WORKFLOW-EXAMPLES.md (hunk 1 above)
Comment (wilke) [reply scripts/WORKFLOW-EXAMPLES.md:1]: No - usually the READE.md files are.

File: QuickStart.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+## Install psi-j
+
+1. Clone repository into your working directory:
+    `git clone https://github.com/ExaWorks/psi-j-python.git`
+
+2. `cd psi-j-python`
+3. `make install`
+
+If you have conda installed you might want to start from a fresh environment:
```
[the remark this answers is not in the document]
Comment (wilke) [reply QuickStart.md]: ok, thanks

File: scripts/pdf2wc4singularity.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kylechard) [reply scripts/pdf2wc4singularity.py:10]: Should we create an exaworks account. We already have docker containers used for CI.

File: QuickStart.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+## Requirements
+- python3.7
+
+## Install psi-j
+
+1. Clone repository into your working directory:
+    `git clone https://github.com/ExaWorks/psi-j-python.git`
+
+2. `cd psi-j-python`
+3. `make install`
```
[the remark this answers is not in the document]
Comment (wilke) [reply QuickStart.md:27]: I am ok changing it. The Makefile contains targets for testing, installing, document creation etc. I did not test with pip install but should be identical.

File: scripts/pdf2wc4singularity.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (hategan) [reply scripts/pdf2wc4singularity.py:10]: > We already have docker containers used for CI.

... and the testing service.

[the remark this answers is not in the document]
Comment (wilke) [reply scripts/pdf2wc4singularity.py:10]: @kylechard There is an ExaWorks account https://hub.docker.com/u/exaworks but I don't know who has access to it.

**Pull request merged**